client/src/pages/Login.js

```diff
@@ -47,13 +50,20 @@ const useStyles = makeStyles((theme) => ({
 
 function Login(props) {
   const classes = useStyles();
+  const history = useHistory();
 
   // State variables
   const [invalidEmail, setInvalidEmail] = useState(false);
-  const [invalidPassword, setInvalidPassword] = useState(false);
-  const [rememberMe, setRememberMe] = useState(false);
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
+  const [invalidLogin, setInvalidLogin] = useState(false);
+  const [rememberMe, setRememberMe] = useState(
+    localStorage.getItem("rememberMe") ? true : false
+  );
```

Comment:
What's the purpose of localStorage? Doesn't the state value do the same thing?

Reply:
This sets the original value of the state variable. If the user chooses "remember me" their credentials will be saved in localStorage, so we want the original value to be whatever is saved in localStorage if it exists.